src/Config.h

```diff
@@ -2,11 +2,13 @@
 
 #include "KafkaW.h"
 #include "uri.h"
+#include "helper.h"
 #include <atomic>
 #include <chrono>
 #include <memory>
 #include <string>
 #include <vector>
+#include <condition_variable>
```

Comment:
You've added an include for `condition_variable` here so you should remove it in the `.cxx` file.